src/web/client/WebExtensionContext.ts

```diff
@@ -52,74 +52,74 @@ class WebExtensionContext {
         currentSchemaVersion: ""
     };
 
-    public getPowerPlatformExtensionContext() {
-        return this.powerPlatformExtensionContext;
+    public getWebExtensionContext() {
+        return this.webExtensionContext;
     }
 
-    public async setPowerPlatformExtensionContext(entityName: string, entityId: string, queryParamsMap: Map<string, string>) {
+    public setWebExtensionContext(entityName: string, entityId: string, queryParamsMap: Map<string, string>) {
```

Comment:
this can be a setter function too

Reply:
https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3116420